packages/gatsby/src/query/__tests__/__snapshots__/query-compiler.js.snap

```diff
@@ -153,37 +158,7 @@ Object {
 }
 `;
 
-exports[`actual compiling errors on double root 1`] = `
-Array [
-  Object {
-    "context": Object {
-      "afterCodeFrame": "  1 | query anotherQueryAndMockFileQuery {
-  2 |   allPostsJson {
-  3 |     #...
-  4 |   }
-  5 |   allPostsJson {
-  6 |     #...
-  7 |   }
-  8 | }",
-      "beforeCodeFrame": "   1 | query mockFileQuery {
-   2 |   allPostsJson {
-   3 |     #...
-   4 |   }
-   5 | }
-   6 |
-   7 | query AnotherQuery {
-   8 |   allPostsJson {
-   9 |     #...
-  10 |   }
-  11 | }",
-      "name": "AnotherQuery",
-      "otherName": "mockFileQuery",
-    },
-    "filePath": "mockFile",
-    "id": "85910",
-  },
-]
-`;
+exports[`actual compiling errors on double root 1`] = `Array []`;
```

Comment:
this indicate that we might have gone too liberal with changes so far - updating snapshots with new fields is one thing, not throwing expected errors is another :)

Reply:
Yeah we  lose error in cases like this probably others too 

```
export const query = graphql`
  query DemoPage {
    site {
      id
    }
  }
  query DemoPage {
    site {
      id
    }
  }
`;
```